test/martha/martha_v3.test.js

```diff
@@ -1124,6 +1106,8 @@ test.skip('martha_v3 should return 500 if access method parsing fails', async (t
 
 /* BT-236 Skip testing access token fetch failure since that is not something this code even attempts with BDC
  * signed URLs turned off. */
```

Comment:
is this comment (and the copy below) stale now that BDC signed URLs are turned on?

Reply:
Sorta. I was leaving it to provide context for the comment that I added. I guess I could have re-written it instead, and I know it could be read from git history. I can make changes if you want.